test/features/searchUser.js

```diff
@@ -0,0 +1,92 @@
+const chaiHttp = require('chai-http');
+const chai = require('chai');
+const mongoose = require('mongoose');
+const app = require('../../server');
+
+const User = mongoose.model('User');
+
+const user = {
```

Comment:
Please move this to a factory folder where it can be called by multiple test files.